sharktank/sharktank/ops/default_impls.py

```diff
@@ -707,6 +707,25 @@ def split_default(
     return torch.split(unbox_tensor(tensor), split_size_or_sections, dim)
 
 
+@swiglu.override(Tensor)
+def swiglu_default(
+    x: Tensor, *, alpha: float = 1.702, limit: float | None = None
+) -> Tensor:
+    x = unbox_tensor(x)
+    if x.size(-1) % 2 != 0:
+        raise ValueError(f"SwiGLU expects even last dim, got {x.size(-1)}")
+
+    x_glu = x[..., ::2]
+    x_lin = x[..., 1::2]
```

Comment:
Nit: can you add a comment to the effect of `pytorch indexing allows for start:stop:step notation`